sbnci/PlottingScripts/PlotTotalPE.C

```diff
@@ -0,0 +1,141 @@
+#define PlotTotalPE_cxx
```

Comment:
So I might be being thick here. Does this script actually get used as part of the validation process? I think it may just be a previous version for plotting PDS stuff that's no longer needed?

Reply:
you are correct again!